tokio/src/runtime/basic_scheduler.rs

```diff
@@ -69,23 +64,23 @@ struct Tasks {
 /// A remote scheduler entry.
 ///
 /// These are filled in by remote threads sending instructions to the scheduler.
-enum Entry {
+enum RemoteMsg {
     /// A remote thread wants to spawn a task.
     Schedule(task::Notified<Arc<Shared>>),
```

Comment:
was the intention behind leaving this an enum just to keep the diff smaller? it seems like it _could_ be changed to a `Schedule` struct, and just making the run queue a `VecDeque<Schedule>`...